### PR TITLE
Delete some Win9x-specific code and comments

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: src/coreclr/debug/di/publish.cpp [hunk 1]
```diff
@@ -81,7 +81,7 @@ BOOL GetAllProcessesInSystem(DWORD *ProcessId,
 
     } while ((succ == TRUE) && (iIndex < (int)dwArraySize));
 
-    // I would like to know if we're running more than 512 processes on Win95!!
+    // I would like to know if we're running more than 512 processes!!
```
Comment (stephentoub) [inline src/coreclr/debug/di/publish.cpp:84]: Who is "I" and why do they want to know that? Strange comment :-)

**Review by stephentoub — APPROVED** [review]

File: src/coreclr/debug/di/publish.cpp (hunk 1 above)
Comment (jkotas) [inline src/coreclr/debug/di/publish.cpp:84]:
```suggestion
```

Comment (jkotas) [reply src/coreclr/debug/di/publish.cpp]: gone

**Review by AaronRobinsonMSFT — APPROVED** [review]

**Comment by jkotas** [comment]
Test failures are #65914, #67816, #67349

**Pull request merged**